FakeServer/Controllers/DynamicController.cs

```diff
@@ -54,20 +54,23 @@ public void UpdateAllData([FromBody]string value)
         /// <remarks>
         /// Add filtering with query parameters. E.q. /api/users?age=22&amp;name=Phil (not possible with Swagger)
         ///
-        /// Optional parameter names skip/take and offset/limit:
+        /// Optional parameter names skip/take and offset/limit and page/per_page:
```

Comment:
I think we could change functionality a little, so this would only allow skip/take, offset/limit or page/per_page combinations. Before mixing skip/take and offset/limit was possible, but it shouldn't have been. 

You can just add new method to QueryHelpers, like IsQueryValid and call it before GetCollection. If it returns false, return a BadRequest.

Reply:
I added in a method for validation as you suggested.